content/src/thumbnail.js

```diff
@@ -61,12 +78,13 @@ function getImageInfo(canvas) {
 
   // Find the longer edge and make it a square.
   var longerEdge;
+  var diff = Math.abs((imageWidth - imageHeight) / 2);
```

Comment:
While you're thinking through these cases
- Do you think we should adjust things so that we never grow an image if there are fewer than 128 pixels?  To me it feels like we should shrink but never grow to make a thumbnail.
- Should there be special handling when there are no pixels at all on the canvas?  (e.g., turtle is hidden, there is no output)